### ISSUE TITLE
After creating an org, unable to add any members (demo site)

### ISSUE THREAD

**Issue by dpalomino** [Cadasta/cadasta-platform#1232]
### Steps to reproduce the error

This is reproducible in demo and not in staging, as far as I've seen. But it seems a serious issue so I think it's worth to take a look

1. Create a new org
2. Try to add a new member
3. A permission denied message is obtained when clicking on "members" panel

Then I logged out and in, and: 
4. I was able to see the org members
5. When trying to add a new member clicking on Add, a ton of permission denied messages was shown

Then I logged out and in again and this time I was able to add new members. 

I've been able to reproduce this a couple of times in demo (timestamp around 20170307190000 GMT+1). Not sure @amplifi if you can take a look to the log. I think we should know what it is happening to be sure that this is not going to happen with the new release, and potentially do a quick fix in demo as well.

**Comment by wonderchook** [comment]
@dpalomino are you clicking on the members panel to add a new member of on the down error next to "Add project"?

**Comment by dpalomino** [comment]
@wonderchook, I was clicking on the "Members" icon on the left first, and then the "+Add" icon on top right corner to add a new member and then I got the permission denied messages. But I am assuming that the behaviour will be the same when selecting the "Add org member"

Behaviour is not being always the same, now I can reproduce steps 1 to 3 with a new org, but after logging out and in I am now able to successfully add new members. 

See video here: 
https://screencast.com/t/RJ5NkLKHq

**Comment by dpalomino** [comment]
As it is only happening with "dpalomino" account, for now decreasing priority to medium.

**Comment by SteadyCadence** [comment]
I have experienced this issue as well with 15 orgs.

**Comment by wonderchook** [comment]
@SteadyCadence so it is consistent now with your account?

**Comment by SteadyCadence** [comment]
yes, I believe so. I forgot that I can logout/login to add members.  So this can be low priority.

**Comment by wonderchook** [comment]
@dpalomino what happens if you logout/login to add members?

On production neither of you have nearly as many orgs associated with you, right?

**Comment by SteadyCadence** [comment]
Correct.  And partners typically set up their own account on production.

**Comment by wonderchook** [comment]
@SteadyCadence would it be possible to try removing a bunch of orgs from that account. Just to see if it starts working again?

**Comment by SteadyCadence** [comment]
Sure.  I will work on that tomorrow when I have some more time.
